chemicaltoolbox/openbabel/ob_remIons.py

```diff
@@ -15,20 +16,25 @@ def parse_command_line():
     parser.add_argument('-iformat', default='sdf', help='input file format')
     parser.add_argument('-i', '--input', required=True, help='input file name')
     parser.add_argument('-o', '--output', required=True, help='output file name')
+    parser.add_argument('-idx', default=False, action='store_true', help='should output be an indexed text table? works only for inchi/smiles, otherwise is ignored')
     return parser.parse_args()
 
 
 def remove_ions(args):
-    outfile = pybel.Outputfile(args.iformat, args.output, overwrite=True)
-    for mol in pybel.readfile(args.iformat, args.input):
+    outfile = open(args.output, 'w')
```

Comment:
Maybe a `with` block here?

Reply:
I thought about having it in `with` block but then removed it because it did not seem necessary. Do you have some point against it, why it would be better with `with` block? I am not exactly python expert so I am happy to learn :)

Reply:
The with block ensures the file gets closed properly, here you are just opening it. See here: https://www.python-course.eu/python3_file_management.php#Reading-and-Writing-Files-in-Python

Reply:
Ok, point made, I will use `with`, thanks for noticing :)